### PR TITLE
Added LSTM Univariate Time series analysis

### PR DESCRIPTION
**Pull request by MuLx10**
Revised univariate LSTM Time series.
An example of using Recurrent Neural Network (RNN) to make forecasts on a time series of international airline passengers using a simple LSTM neural network using mlpack.

Dataset
A CSV containing a time series data set of international airline passengers; monthly totals in thousands. Jan 49 – Dec 60

Results for 10 iterations:
```bash
1 - MeanSquaredError := 0.146075
2 - MeanSquaredError := 0.144882
3 - MeanSquaredError := 0.09501
4 - MeanSquaredError := 0.0875479
5 - MeanSquaredError := 0.0836975
6 - MeanSquaredError := 0.0796567
7 - MeanSquaredError := 0.0804368
8 - MeanSquaredError := 0.0803483
9 - MeanSquaredError := 0.0809061
10 - MeanSquaredError := 0.076797
```

### CODE REVIEW

**Comment by zoq** [comment]
What results do you expect, do you think we will see anything better as 0.076797?

**Comment by MuLx10** [comment]
I didn't understand. Do you want me to run it for more iterations?

**Comment by zoq** [comment]
Not necessarily, I was just wondering what results we could expect from the dataset and model, maybe there are some baseline results for the dataset? That give us some insights if ``0.076797``is good or not. For example, we have a pretty good idea what results we like to see if we use the mnist dataset; since I haven't worked with the passengers dataset, I don't really know what results are good or not.

**Comment by zoq** [comment]
Happy to clarify what I wrote above, if you like.

**Comment by MuLx10** [comment]
I guess finding another known dataset should do.

**Comment by zoq** [comment]
Maybe, do you have a reference to the dataset you used here?

**Comment by MuLx10** [comment]
Similar to https://www.kaggle.com/chirag19/air-passengers.

**Comment by zoq** [comment]
Thanks for the link, personally I would probably use another dataset as you already suggested, that  provides more data, the dataset is somewhat small. But I think this is an interesting dataset so I would probably keep it for the tutorial section.

**Comment by MuLx10** [comment]
@zoq Please let me know if [this](https://openmv.net/info/distillate-flow) dataset will be good?

**Comment by zoq** [comment]
Looks good, also what do you think about: https://openmv.net/info/electricity-usage. Maybe we should look into both?

**Comment by MuLx10** [comment]
> Looks good, also what do you think about: https://openmv.net/info/electricity-usage. Maybe we should look into both?

I have added for electricity usage dataset.

**Comment by zoq** [comment]
Do you like to provide the results for the airline passenger dataset as well? If you like we could do both, if not that's fine for me as well, just let me know.

**Comment by MuLx10** [comment]
> Do you like to provide the results for the airline passenger dataset as well? If you like we could do both, if not that's fine for me as well, just let me know.

Should I add that in tutorials?

**Comment by zoq** [comment]
Your call, personally I think it would be nice to have another dataset, and I guess since you already have the result it would be straightforward.

**Comment by MuLx10** [comment]
> Your call, personally I think it would be nice to have another dataset, and I guess since you already have the result it would be straightforward.

Sure.

File: LSTM/TimeSeries-Univariate/tutorial.txt [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+6 - MeanSquaredError := 0.0796567
+7 - MeanSquaredError := 0.0804368
+8 - MeanSquaredError := 0.0803483
+9 - MeanSquaredError := 0.0809061
+10 - MeanSquaredError := 0.076797
+....
+
+MSE Plot
+
+![Plot](https://user-images.githubusercontent.com/23444642/62627150-6e3f4400-b946-11e9-9a24-1542e564da42.png)
```
Comment (zoq) [inline LSTM/TimeSeries-Univariate/tutorial.txt]: This looks really good to me, thanks for putting this together.

**Comment by MuLx10** [comment]
> For my side this is ready to go, is there anything you like to add?

I am good to go.

**Comment by zoq** [comment]
Great, thanks for implementing another model.

**Pull request merged**